packages/metro/src/DeltaBundler/getTransformCacheKey.js

```diff
@@ -42,6 +43,7 @@ function getTransformCacheKey(opts: {|
         'metro-cache',
         VERSION,
         opts.cacheVersion,
+        path.resolve(path.join(__dirname, '../../..'), opts.projectRoot),
```

Comment:
I don't think we can assume that `../../..` is meaningful here, as the `metro` can be installed and used in different ways/locations. Even if we wanted to resolve `projectRoot` to an absolute path, we would have to do it in a way that is always correct.

Reply:
can we use `process.cwd()` instead?